app/jobs/get_suggestions_job.rb

```diff
@@ -0,0 +1,93 @@
+class GetSuggestionsJob < ApplicationJob
+  queue_as :default
+
+  def perform(job_status)
+    @job_id = job_status.job_id
```

Comment:
What's the performance on passing the `SuggestionsJobStatus` instance through to the background job? Should we only pass the id and then re-find it to save on memory?